gulpfile.js

```diff
@@ -67,6 +69,16 @@ gulp.task('lib', function() {
 gulp.task('flow', function() {
   return gulp
     .src(paths.lib.src)
+    .pipe(gulpModuleMap(moduleMapOpts))
+    .pipe(gulpStripProvidesModule())
```

Comment:
Hmm, I don't think we can have it both ways, though. Haste wants non-relative requires, Node wants relative, right?